torch/onnx/_globals.py

```diff
@@ -31,6 +31,13 @@ def __init__(self):
         self.operator_export_type: Optional[_C_onnx.OperatorExportTypes] = None
         self.onnx_shape_inference: bool = True
 
+        # Internal feature flags
+        self.runtime_type_check: Optional[bool] = None
```

Comment:
Would an enum be better if value `None` represents nontrivial behavior?

Reply:
Done